cmd/cmd.go

```diff
@@ -53,37 +50,48 @@ func PrintUsage() {
 	fmt.Println()
 }
 
-func defaultHomeDir() (string, error) {
+func defaultSkybinHome() (string, error) {
 	user, err := user.Current()
 	if err != nil {
 		return "", err
 	}
 	return path.Join(user.HomeDir, ".skybin"), nil
 }
 
-func findHomeDir() (string, error) {
+func findSkybinHomedir() (string, error) {
 	homedir := os.Getenv("SKYBIN_HOME")
 	if len(homedir) == 0 {
 		var err error
-		homedir, err = defaultHomeDir()
+		homedir, err = defaultSkybinHome()
 		if err != nil {
 			return "", err
 		}
 	}
 	return homedir, nil
 }
 
-func getRenterClient() (*renter.Client, error) {
-	homedir, err := findHomeDir()
-	if err != nil {
-		return nil, err
+func findRenterHomedir() (string, error) {
+	homedir := os.Getenv("SKYBIN_RENTER_HOME")
+	if len(homedir) == 0 {
+		var err error
+		homedir, err = findSkybinHomedir()
+		if err != nil {
+			return "", err
+		}
+		homedir = path.Join(homedir, "renter")
 	}
+	return homedir, nil
+}
 
-	var config renter.Config
-	err = util.LoadJson(path.Join(homedir, "renter", "config.json"), &config)
-	if err != nil {
-		return nil, fmt.Errorf("Cannot load renter config. Error: %s", err)
+func findProviderHomedir() (string, error) {
+	homedir := os.Getenv("SKYBIN_PROVIDER_HOME")
+	if len(homedir) == 0 {
+		var err error
```

Comment:
Is there a reason you're declaring error here? Does it have to do with that one bug you hit?

Reply:
It is because I would like to reassign to `homedir`, but using the `:=` assignment operator would create a new variable called homedir within the scope of the if block.